packages/models/index.ts

```diff
@@ -1,2 +1,35 @@
+import { K8sModel } from '@openshift-console/dynamic-plugin-sdk/lib/api/common-types';
+
 export * from './mock';
 export * from './noobaa';
+
+
+export const PersistentVolumeClaimModel: K8sModel = {
+  label: 'PersistentVolumeClaim',
+  // t('public~PersistentVolumeClaim')
+  labelKey: 'public~PersistentVolumeClaim',
+  apiVersion: 'v1',
+  plural: 'persistentvolumeclaims',
+  abbr: 'PVC',
+  namespaced: true,
+  kind: 'PersistentVolumeClaim',
+  id: 'persistentvolumeclaim',
+  labelPlural: 'PersistentVolumeClaims',
+  // t('public~PersistentVolumeClaims')
+  labelPluralKey: 'public~PersistentVolumeClaims',
+};
+
+export const SecretModel: K8sModel = {
+  apiVersion: 'v1',
+  label: 'Secret',
+  // t('plugin__odf-console~Secret')
+  labelKey: 'plugin__odf-console~Secret',
+  plural: 'secrets',
+  abbr: 'S',
+  namespaced: true,
+  kind: 'Secret',
+  id: 'secret',
+  labelPlural: 'Secrets',
+  // t('plugin__odf-console~Secrets')
+  labelPluralKey: 'plugin__odf-console~Secrets',
+};
```

Comment:
`index` file is only for exporting other files...
example:
export * from './mock';
export * from './noobaa';

create a new file for above models and export that file from here... same while importing these models check how we do so from `odf-console` repo...